src/components/game/ScreenshotPage.js

```diff
@@ -55,38 +104,46 @@ export default function ScreenshotPage({
 }) {
   const classes = useStyles()
   const globalContext = useContext(Context)
-  const playersGettingPoints =
-    winningTeam === 'whiteTeam'
-      ? [winningJudge, ...whiteTeam]
-      : winningTeam === 'blueTeam'
-      ? [winningJudge, ...blueTeam]
-      : []
-  const wordGuessed =
-    winningTeam === 'whiteTeam'
-      ? whiteTeamWord
-      : winningTeam === 'blueTeam'
-      ? blueTeamWord
-      : 'No one'
-  const winningTeamDisplay =
-    winningTeam === 'whiteTeam' ? 'White Team' : winningTeam === 'blueTeam' ? 'Blue Team' : 'No one'
+  const [openRanking, setOpenRanking] = useState(false)
+  const handleCloseRanking = () => {
```

Comment:
This is a nit, but it seems a bit unnecessary to create a wrapper function for the setter function. `setOpenRanking` seems explicit enough to me, whereas `handleCloseRanking` sounds like it would have some additional logic (setting a field, submitting a request, handling data, etc). I would just use the explicit setter function (and if you need to invoke it, just use `() => setOpenRanking()`). This is more personal preference though.